packages/core/code-runner/package.json

```diff
@@ -25,28 +25,29 @@
     "url": "https://github.com/embark-framework/embark.git"
   },
   "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "embark-collective": {
-    "build:node": true
+    "build:node": true,
+    "typecheck": true
   },
   "scripts": {
     "_build": "npm run solo -- build",
+    "_typecheck": "npm run solo -- typecheck",
```

Comment:
Until we have collective actions for all of typecheck, lint, build, and test, it makes sense to keep `"ci"` / `"qa"` working the way they do presently, i.e. lerna works through the packages one by one running the series of tasks in each of them. So there needs to be some `package.json` script that a package's `"qa"` script can invoke for each task in that package. If `_build` were `build` and/or `_typecheck` were `typecheck` there would be a collision with how lerna and `embark-collective` work, so I named them with the leading `_`. Note that they in turn invoke the `solo` script, which uses `embark-solo` as a way to launch the collective task but scoped only to that package, i.e. we don't a package-level *implementation* of the task and a collective implementation, just the collective one, but I designed `embark-collective` so that collective actions can be scoped.

Reply:
Once the other collective actions have been implemented, all of the `"ci"` and `"qa"` scripts will be removed from individual packages that are enabled for the collective (which is almost all of them except `packages/cockpit/ui`).